docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -158,19 +170,23 @@ Three steps to start an experiment
 
 All the code above is already prepared and stored in :githublink:`examples/trials/mnist-pytorch/ <examples/trials/mnist-pytorch>`.
 
+
+Step 4: Launch the Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 Linux and macOS
-^^^^^^^^^^^^^^^
+***************
 
-Run the **config.yml** file from your command line to start an MNIST experiment.
+Run the **config.yml** file from your command line to start the experiment.
 
 .. code-block:: bash
 
    nnictl create --config nni/examples/trials/mnist-pytorch/config.yml
 
 Windows
-^^^^^^^
+*******
 
-Run the **config_windows.yml** file from your command line to start an MNIST experiment.
+Run the **config_windows.yml** file from your command line to start the experiment.
```

Comment:
Ooops, I removed config_windows.yml. I thought it's suggesting NNI behaves differently on Windows and Linux, which is not the case.
If it makes doc or quick start hard or complex, please add it back. Just copy config.yml and change `python3` to `python`.

Reply:
hahaha, a user asked in the WeChat group why there is no `config_windows.yml` example for v2.3, so I add it back?

Reply:
Another choice is to tell the users to update the config. I don't know which is better.

Reply:
I modified all the config files in quickstart to `config_detailed.yml`, which including the inline search space and swap the search space part and the trial code part so that users can config the experiment directly in the YAML file after writing the search space.